codeflash/api/aiservice.py

```diff
@@ -182,6 +182,7 @@ def optimize_python_code_line_profiler(  # noqa: D417
             "python_version": platform.python_version(),
             "experiment_metadata": experiment_metadata,
             "codeflash_version": codeflash_version,
+            "lsp_mode": is_LSP_enabled(),
```

Comment:
Q: What changes in aiservices are we thinking to change for LSP server?
No of candidates and tests?